examples/acquiring_data/downloading_hmi.py

```diff
@@ -32,15 +35,34 @@
 # don't provide a path it will download the file into the sunpy data directory.
 # The output provides the path of the downloaded files.
 # The result can be from several data clients, so we have to index the client first
-# client and then index the file.
+# and then index the file.
 
 # Slice the first record returned by the first client.
-downloaded_file = Fido.fetch(result[0, 0])
+downloaded_file = Fido.fetch(result[0])
 print(downloaded_file)
 
 ###############################################################################
-# Now load it into a map and plot it
+# Now load it into a map and plot it.
+# We see that solar North is pointed down instead of up in this image, which is
+# indicated by the coordinates (that range from positive to negative, rather
+# than negative to positive).
 hmi_map = sunpy.map.Map(downloaded_file[0])
 fig = plt.figure()
 hmi_map.plot()
 plt.show()
+
+###############################################################################
+# Now rotate the image such that solar North is pointed up. 
+# We have to do this because the HMI instrument is mounted upside-down 
+# relative to the AIA instrument on the SDO satellite, which means most 
+# of the images are taken with solar North pointed up. 
+# The roll angle of the instrument is reported in the FITS header
+# keyword `CROTA2` (see Figure 17 of 
+# `Couvidat et al. (2016)<https://dx.doi.org/10.1007/s11207-016-0957-3>`_, 
```

Comment:
```suggestion
# `Couvidat et al. (2016) <https://dx.doi.org/10.1007/s11207-016-0957-3>`_, 
```

I think this is what is causing the docs build to fail?

Reply:
Ugh, sorry Will!

Reply:
Wow that actually fixed it 😆